pylzy/tests/api/v1/test_calls.py

```diff
@@ -0,0 +1,242 @@
+from typing import Optional, List, Tuple
+from unittest import TestCase, skip
+
+from pydantic import ValidationError
+
+from lzy.api.v1 import Lzy, op
+from lzy.api.v1.signatures import FuncSignature
+from lzy.api.v1.utils.proxy_adapter import materialized
+from tests.api.v1.mocks import RuntimeMock, StorageRegistryMock
+
+
+@op(description="aaa")
```

Comment:
Where is this description used in tests?

Reply:
Removed it :)